src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -577,6 +578,8 @@ InputProcessor::MaxFields InputProcessor::findMaxFields(EnergyPlusData &state, j
         }
     } else {
         auto const &legacy_idd_fields = legacy_idd["fields"];
+        // start with at least min_fields as the number of fields
+        maxFields.max_fields = min_fields;
```

Comment:
`max_fields` set initially to `min_fields` so all fields up to that point will be filled with default values if they are not present in the epJSON input.